doc/source/_static/js/whatsnew_links.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+	const links = document.getElementsByTagName('a');
+	for (let i = 0; i < links.length; i++) {
```

Comment:
Slightly more stylistically speaking:

Thoughts about using more lambda's and array comprehensions (i.e. `.forEach` instead of this)?

Using `function <name>` is perfectly fine.  Personally perception though is that `const <name> =...` is a little more modern, but that's just me.

Reply:
I've rewritten the functions in a moodern way now, it should be a lot more concise now!